### PR TITLE
feat(24.10): add slices for Redis and dependencies

### PR DESCRIPTION
**Pull request by norrisjeremy**
# Proposed changes
feat(24.10): add slices for Redis and dependencies

## Related issues/PRs
#426

## Checklist
* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have tested my changes ([see how](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr))
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

## Additional Context
```shell
$ chisel cut --release chisel-releases/ --root foo redis-server_bins
2024/12/07 09:04:55 Processing chisel-releases/ release...
2024/12/07 09:04:56 Selecting slices...
2024/12/07 09:04:56 Fetching ubuntu 24.10 oracular suite details...
2024/12/07 09:04:56 Release date: Fri, 11 Oct 2024 10:20:59 UTC
2024/12/07 09:04:56 Fetching index for ubuntu 24.10 oracular main component...
2024/12/07 09:04:56 Fetching index for ubuntu 24.10 oracular universe component...
2024/12/07 09:04:56 Fetching ubuntu 24.10 oracular-security suite details...
2024/12/07 09:04:56 Release date: Sat, 07 Dec 2024 11:18:07 UTC
2024/12/07 09:04:56 Fetching index for ubuntu 24.10 oracular-security main component...
2024/12/07 09:04:56 Fetching index for ubuntu 24.10 oracular-security universe component...
2024/12/07 09:04:56 Fetching ubuntu 24.10 oracular-updates suite details...
2024/12/07 09:04:56 Release date: Sat, 07 Dec 2024 13:33:42 UTC
2024/12/07 09:04:56 Fetching index for ubuntu 24.10 oracular-updates main component...
2024/12/07 09:04:56 Fetching index for ubuntu 24.10 oracular-updates universe component...
2024/12/07 09:04:56 Fetching pool/main/b/base-files/base-files_13.3ubuntu6_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/g/gcc-14/gcc-14-base_14.2.0-4ubuntu2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/i/init-system-helpers/init-system-helpers_1.66ubuntu1_all.deb...
2024/12/07 09:04:56 Fetching pool/main/g/gcc-14/libatomic1_14.2.0-4ubuntu2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/g/glibc/libc6_2.40-1ubuntu3_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/libc/libcap2/libcap2_2.66-5ubuntu3_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/g/gcc-14/libgcc-s1_14.2.0-4ubuntu2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/universe/j/jemalloc/libjemalloc2_5.3.0-2build1_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/g/gcc-14/libstdc++6_14.2.0-4ubuntu2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/universe/libl/liblzf/liblzf1_3.6-4_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/o/openssl/libssl3t64_3.3.1-2ubuntu2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/libz/libzstd/libzstd1_1.5.6+dfsg-1_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/z/zlib/zlib1g_1.3.dfsg+really1.3.1-1ubuntu1_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/s/systemd/libsystemd0_256.5-2ubuntu3.1_amd64.deb...
2024/12/07 09:04:56 Fetching pool/universe/r/redis/redis-server_7.0.15-1build2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/universe/r/redis/redis-tools_7.0.15-1build2_amd64.deb...
2024/12/07 09:04:56 Fetching pool/main/s/sysvinit/sysvinit-utils_3.08-6ubuntu3_amd64.deb...
2024/12/07 09:04:56 Extracting files from package "base-files"...
2024/12/07 09:04:56 Extracting files from package "gcc-14-base"...
2024/12/07 09:04:56 Extracting files from package "init-system-helpers"...
2024/12/07 09:04:56 Extracting files from package "libatomic1"...
2024/12/07 09:04:56 Extracting files from package "libc6"...
2024/12/07 09:04:57 Extracting files from package "libcap2"...
2024/12/07 09:04:57 Extracting files from package "libgcc-s1"...
2024/12/07 09:04:57 Extracting files from package "libjemalloc2"...
2024/12/07 09:04:57 Extracting files from package "libstdc++6"...
2024/12/07 09:04:57 Extracting files from package "liblzf1"...
2024/12/07 09:04:57 Extracting files from package "libssl3t64"...
2024/12/07 09:04:57 Extracting files from package "libzstd1"...
2024/12/07 09:04:57 Extracting files from package "zlib1g"...
2024/12/07 09:04:57 Extracting files from package "libsystemd0"...
2024/12/07 09:04:57 Extracting files from package "redis-server"...
2024/12/07 09:04:57 Extracting files from package "redis-tools"...
2024/12/07 09:04:57 Extracting files from package "sysvinit-utils"...
$ sudo mkdir -p foo/run/redis foo/var/lib/redis foo/var/log/redis
$ sudo chroot foo redis-server /etc/redis/redis.conf
$ sleep 1
$ sudo chroot foo redis-cli PING
PONG
$ sudo kill -TERM $(cat foo/run/redis/redis-server.pid)
```

### CODE REVIEW

File: slices/redis-server.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+slices:
+  bins:
+    essential:
+      - init-system-helpers_bins
+      - redis-server_config
+      - redis-tools_bins
+      - sysvinit-utils_bins
+    contents:
+      /etc/init.d/redis-server:
```
Comment (cjdcordeiro) [inline slices/redis-server.yaml:14]: (same as https://github.com/canonical/chisel-releases/pull/426#discussion_r1881749503)

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: slices/redis-server.yaml (hunk 1 above)
Comment (norrisjeremy) [reply slices/redis-server.yaml:14]: See https://github.com/canonical/chisel-releases/pull/426#discussion_r1881781849.

**Comment by norrisjeremy** [comment]
Does anybody know why the ARM64 spread tests failed?

**Review by cjdcordeiro — APPROVED** [review]
LGTM. Thanks. Merging without a 2nd approval since it's a forward port of the already approved and merged #426

**Pull request merged**